content/Hardware Support/UNO/Update-Modulino-firmware.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Update Modulino firmware"
+id: 
+---
+
+Learn how to update the firmware of a [Modulino node](https://store-usa.arduino.cc/products/plug-and-make-kit?srsltid=AfmBOopomr-DZVLRzl_VhXSSemiNa75qfABCHjqhvLQs7vz9_2PLgQN9#:~:text=technology%20%E2%80%93%20and%20yours.-,What%20are%20Modulino%C2%AE%3F,-Modulino%C2%AE%20are%20sensors).
+
+Updating the firmware may be necessary to resolve issues with Modulino nodes.
+
+In this article:
+
+- [Update Modulino firmware in Arduino IDE 2](#update-modulino-firmware-in-arduino-ide-2)
+- [Update Modulino firmware in Arduino Cloud Editor](#update-modulino-firmware-in-arduino-cloud-editor)
+
+---
+
+<a id="ide"></a>
+
+## Update Modulino firmware in Arduino IDE 2
+
+1. Connect the Modulino to your Arduino UNO R4 WiFi.
+
+2. Connect the board to your computer and open Arduino IDE 2.
+
+3. [Go to the Library Manager](https://support.arduino.cc/hc/en-us/articles/5145457742236-Add-libraries-to-Arduino-IDE) and install the [Modulino](https://www.arduino.cc/reference/en/libraries/modulino/) and [ArduinoGraphics](https://www.arduino.cc/reference/en/libraries/arduinographics/) libraries.
+
+4. In the top menu bar, open **File > Examples > Modulino > Utililities > FirmwareUpdater**.
+
+5. Click ![Upload button](img/symbol_upload2.png) **Upload** to upload the sketch to your Arduino UNO R4 WiFi.
+
+6. The firmware update is complete when the sketch is successfully uploaded to the board.
```

Comment:
I gave it some thought, and I think it would be good to mention that the LED matrix here. This is ensures that the process is complete and avoids the risk of confusion about the meaning of the `PAS` text.

Quick suggestion below, feel free to do whatever (but make the equivalent change [here].(https://github.com/arduino/help-center-content/pull/422/files#diff-192493aa6768dcc30885ad278b65a87b516a88b281fb64126b2b3a56366204baR55))

```suggestion
6. The firmware update is complete when the sketch is successfully uploaded to the board and the text `PAS` is displayed on the LED matrix.
```